study.md

```diff
@@ -21,48 +21,83 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Write the Ruby code that takes the integer `700` and returns the string `"007"`.
 
 ```ruby
-# your answer here
+700.to_s
 ```
 
+[Poignant Guide](http://poignant.guide/book/chapter-3.html)
+
 ## Array Manipulation
 
 Write the Ruby code that takes the following array, `[23, 56, 3, 7]`, and
 manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+[23, 56, 3, 7].sort.reverse
```

Comment:
Since we want to modify the original array, the calls to `sort` and `reverse` should be `sort!` and `reverse!`, respectively.